test/Chainweb/Test/Mempool.hs

```diff
@@ -113,38 +122,69 @@ mempoolTestCase name test (MempoolWithFunc withMempool) =
     tout m = timeout 60000000 m >>= maybe (fail "timeout") return
 
 
-mempoolProperty :: TestName
-                -> PropertyM IO a
-                -> (a -> MempoolBackend MockTx -> IO (Either String ()))
-                -> MempoolWithFunc
-                -> TestTree
+mempoolProperty
+    :: TestName
+    -> PropertyM IO a
+    -> (a -> InsertCheck -> MempoolBackend MockTx -> IO (Either String ()))
+    -> MempoolWithFunc
+    -> TestTree
 mempoolProperty name gen test (MempoolWithFunc withMempool) = testProperty name go
   where
     go = monadicIO (gen >>= run . tout . withMempool . test
                         >>= either fail return)
 
     tout m = timeout 60000000 m >>= maybe (fail "timeout") return
 
-testStartup :: MempoolBackend MockTx -> IO ()
-testStartup = const $ return ()
+testStartup :: InsertCheck -> MempoolBackend MockTx -> IO ()
+testStartup = const $ const $ return ()
```

Comment:
Does `mempty` work here?

Reply:
What are you suggesting

Reply:
I think `const $ mempty` or just `mempty` will compile.